tests/test_comp_regressor.py

```diff
@@ -0,0 +1,80 @@
+import os
+import unittest
+import warnings
+from pathlib import Path
+
+# prevent excessive warning logs 
+warnings.filterwarnings('ignore')
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+import tensorflow as tf
+import pandas as pd
+import numpy as np
+import random
+
+from sklearn.model_selection import train_test_split
+from finetune import MultiFieldClassifier, MultiFieldRegressor, Regressor
+from finetune.config import get_config
+from finetune.datasets import generic_download
+from finetune.comparison_regressor import ComparisonRegressor
+
+class TestModel(unittest.TestCase):
```

Comment:
This TestModel is what shows up in the test logs so it needs to be something more expressive